Model/articles.model.js

```diff
@@ -2,7 +2,13 @@ const db = require('../db/connection')
 
 exports.selectArticleById = (article_id) => {
 
-    return db.query('SELECT * FROM articles WHERE article_id = $1', [article_id]).then(results => {
+    return db.query(`SELECT articles.title, articles.topic, articles.author, articles.body, articles.created_at, articles.votes, CAST(COUNT(comments.article_id) AS int) as comment_count
+    FROM articles 
```

Comment:
A sneaky way to select all columns just from the articles table is to use the syntax `select articles.*`
This means you don't need to manuallywrite every single column from the articles table, even when we have joined another table